templates/docs/examples/patterns/navigation/reduced.html

```diff
@@ -0,0 +1,201 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Navigation / Reduced{% endblock %}
+
+{% block standalone_css %}patterns_navigation{% endblock %}
+
+{% block content %}
+<header id="navigation" class="p-navigation--reduced is-dark">
+  <div class="p-navigation__row--25-75">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="#">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
+          </div>
+          <span class="p-navigation__logo-title">Canonical Ubuntu</span>
+        </a>
+      </div>
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item">
+          <a href="#/search" class="js-search-button p-navigation__link--search-toggle" aria-label="Search"></a>
+        </li>
+        <li class="p-navigation__item">
+          <a href="#/navigation" class="js-menu-button p-navigation__link">Menu</a>
+        </li>
+      </ul>
+    </div>
+    <nav class="p-navigation__nav js-show-nav" aria-label="Categories">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item--dropdown-toggle" role="menuitem" id="products">
```

Comment:
These top-nav dropdown items don't have proper accessibility attributes to inform screen readers that they are expanded or collapsed. When I go through these with a screen reader it's not apparent that these are dropdowns.

Reply:
We are just reusing the existing Vanilla navigation dropdown. I suppose it is the case also in the standard nav?

If so, I'd consider it a separate issue, outside of the scope of this PR, but thanks for flagging it.

Reply:
This also occurs in existing Vanilla dropdown. Filing a separate issue for that.